hawc/apps/lit/templates/lit/reference_search.html

```diff
@@ -2,7 +2,7 @@
 
 {% block content %}
 <h2>References ({{page_obj.paginator.count}} found)</h2>
-{% include 'common/filter_list.html' with plural_object_name='references' %}
```

Comment:
delete filter_list.html?

Reply:
it's still used for filter lists that haven't been updated to expandable/inline